src/sentry/integrations/github/issues.py

```diff
@@ -28,10 +32,15 @@ def after_link_issue(self, external_issue, **kwargs):
 
         comment = data.get("comment")
         if comment:
-            try:
-                client.create_comment(repo=repo, issue_id=issue_num, data={"body": comment})
-            except ApiError as e:
-                raise IntegrationError(self.message_from_error(e))
+            with sentry_sdk.start_span(
```

Comment:
Creating a Span here was the only way that I was able to get the status codes to show up correctly:
 
<img width="889" alt="Screen Shot 2020-03-26 at 2 57 26 PM" src="https://user-images.githubusercontent.com/15368179/77700784-253e0d80-6f72-11ea-8856-121e85d9f23c.png">

And then when viewing the trace for an individual  `integration.http_response.github` with status code of `403` it links only back to the `create_comment` transaction, not the original frontend request to 'link a github issue':

<img width="900" alt="Screen Shot 2020-03-26 at 2 53 27 PM" src="https://user-images.githubusercontent.com/15368179/77700747-10fa1080-6f72-11ea-8af7-4a99833541ea.png">


I'm not sure if this is the best way, I could't get it working adding in the Span creation in the `group_integration_details` endpoint for some reason but I might try that again and see @markstory @untitaker

Reply:
If you don't use Span() you should inherit the trace I'd. Or was that not working here?

Reply:
Yeah it didn't inherit the trace id, I don't fully understand why though

Reply:
We don't yet inherit the trace ID for new transactions because we don't really expect you to create nested transactions in the same process.